### PR TITLE
Updating input topic

### PR DESCRIPTION
**Pull request by YoannaIvanova**

### CODE REVIEW

File: en/components/input.md [hunk 1]
```diff
@@ -62,27 +55,11 @@ Every experienced designer uses constraints wisely to limit the user input and a
 
 ## Layout
 
-The Input has rich support for prefix and suffix through text string or icon that can, in certain cases, reduce the input effort for the user: e.g. an @email.com suffix means both less keystrokes and more clarity of expected content, while a clock or calendar prefix may indicate that the Input is suitable to provide time or date as content.
+The Input has rich support for prefix and suffix through text string or icon that can, in certain cases, reduce the input effort for the user: e.g. an @email.com suffix means both less keystrokes and more clarity of expected content.
```
Comment (StefanIvanov) [inline en/components/input.md:58]:
```suggestion
The Input has rich support for prefix and suffix through a combination of text and icons that can, in certain cases, reduce the input effort for the user: e.g. an @email.com suffix means both fewer keystrokes and more clarity of expected content. You can remove the prefix or suffix by setting the respective container to ~No Symbol from the overrides.
```

File: en/components/input.md [hunk 2]
```diff
@@ -62,27 +55,11 @@ Every experienced designer uses constraints wisely to limit the user input and a
...
-## Date Picker and Time Picker Inputs
-
-These two are special types of Input customized for the purposes of date and time selection. They have a consistent structure with the other Inputs, but the layout, which is fixed in a certain way for each state. The icons that appear at the prefix location are set to the Material Icons `calendar-today` and `access-time` and can not be changed via the overrides panel.
-
-<img class="responsive-img" src="../images/input_calendar.png" srcset="../images/input_calendar@2x.png 2x" />
-<img class="responsive-img" src="../images/input_time-picker.png" srcset="../images/input_time-picker@2x.png 2x" />
-
 ## Styling
 
 The Input comes with styling achievable through changing the primary, success, warning, and error colors of your theme in the Styling library.
```
Comment (StefanIvanov) [inline en/components/input.md:65]:
```suggestion
The Input comes with styling flexibility through the colors of its background and bottom line, as well as icon color and text styles of its prefix, suffix, and main areas of content. To change the validation styles, it is recommended to update the respective color variables in the Indigo.Design library. 
```

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

File: en/components/input.md [hunk 3]
```diff
@@ -6,32 +6,32 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
...
+<img class="responsive-img" src="../images/input_hint.png" srcset="../images/input_hint@2x.png 2x" />
+<img class="responsive-img" src="../images/input_searchbar.png" srcset="../images/input_searchbar@2x.png 2x" />
 
-<img class="responsive-img" src="../images/input_line.png" srcset="../images/input_line@2x.png 2x" />
-`line`
+The hint and simple presets offer choice between three distinct types: line style for a more airy look, border style for a more structured perception on solid color backgrounds, and boxed style which is most appropriate when the Input is placed on top of a vivid image to improve the readability of its content.
 
-<img class="responsive-img" src="../images/input_box.png" srcset="../images/input_box@2x.png 2x" />
-`box`
+<img class="responsive-img" src="../images/input_simple.png" srcset="../images/input_simple@2x.png 2x" />
```
Comment (StefanIvanov) [inline en/components/input.md:25]: add below a border and box type

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

**Review by StefanIvanov — APPROVED** [review]

File: en/components/input.md [hunk 4]
```diff
@@ -6,32 +6,34 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
...
-`search`
+Each of these presets can be inserted in an enabled or disabled state.
 
-## Variants
+<img class="responsive-img" src="../images/input_enabled.png" srcset="../images/input_enabled@2x.png 2x" />
+<img class="responsive-img" src="../images/input_disabled.png" srcset="../images/input_disabled@2x.png 2x" />
+
+## Variant
 
 The Input can be used styled in **dark** and light variants to assure good readability and contrast for both lighter and darker backgrounds.
```
Comment (sdimchevski) [inline en/components/input.md:38]:
```suggestion
The Input can be styled in **dark** and light variants to assure good readability and contrast for both lighter and darker backgrounds.
```

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**